security/MemberAuthenticator.php

```diff
@@ -51,7 +51,7 @@ protected static function authenticate_member($data, $form, &$success) {
 		if($asDefaultAdmin) {
 			// If logging is as default admin, ensure record is setup correctly
 			$member = Member::default_admin();
-			$success = !$member->isLockedOut() && Security::check_default_admin($email, $data['Password']);
+			$success = Security::check_default_admin($email, $data['Password']) && !$member->isLockedOut();
```

Comment:
IMO the issue described in #7011 shows that we should code defensively and check that `Member::default_admin()` doesn’t return `null` here too:

```php
$success = Security::check_default_admin($email, $data['Password']);
if ($success && $member && !$member->isLockedOut() {
    return $member;
}
```

Thoughts?